src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,79 @@
+.product-card {
+  box-sizing: border-box;
+  width: $product-card-width;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: $transition-params;
+
+  &__title {
+    display: block;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $main-color;
+    text-decoration: none;
+  }
+
+  &:hover {
+    scale: 1.2;
+    .product-card__title {
+      color: #34568b;
+    }
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+    line-height: 140%;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    align-self: center;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__buy-btn {
+    display: block;
+    max-height: 40px;
+    padding: 12px 0;
+    box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)

Reply:
it's against the rule, or I didn't understand something
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules.html
CSS
Don't use * selector (it impacts performance)

Reply:
Feel free to ask for help in the chat